functions/src/bills/types.ts

```diff
@@ -72,7 +363,8 @@ export const Bill = withDefaults(
     history: BillHistory,
     similar: Array(Id),
     currentCommittee: Optional(CurrentCommittee),
-    city: Optional(String)
+    city: Optional(String),
+    topics: Array(String)
```

Comment:
Should topics be Optional here? Not sure if that changes much.

Reply:
Fair point - I felt that the proper missing state for an array is usually just an empty array, but given that this should only be empty while we're waiting for the LLM to choose topics, `Optional(Array(String))` is probably more intuitive and leaves us a little more information to work with if the LLM screws up.